ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -547,7 +547,7 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
               accessLevel: ws.accessLevel
             }));
         }
-        navigate(['workspaces', workspace.namespace, workspace.id, 'data']);
+        navigate(['workspaces', workspace.namespace, workspace.id, 'about']);
```

Comment:
IMO, workspace edit should route back to the about page because that is where a user starts when they go to edit their research purpose.  plus, that is where they will be able to see the results of their changes.  it doesn't make sense to me for them to be routed to the data tab

Reply:
They can also edit via the nav-bar, so technically we don't know where they started. But this seems reasonable to me.